Assets/UdonSharp/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "com.merlinvr.udonsharp",
   "displayName": "UdonSharp",
-  "version": "1.0.0",
+  "version": "1.0.0-dev-serialization",
```

Comment:
Is the package manager okay with this? I think it yells at you if you don't follow semantic versioning exactly.

Reply:
Yep! There are some things it doesn't like, like I wasn't able to pull a package based on `rel_2022.1.1-package-updates`, but this one it's fine with.
![image](https://user-images.githubusercontent.com/737888/155625709-376cc4c6-c1dd-40bf-8fec-b71f4ad7093c.png)